virtio-devices/src/transport/pci_device.rs

```diff
@@ -888,28 +888,26 @@ impl PciDevice for VirtioPciDevice {
             .push((virtio_pci_bar_addr, CAPABILITY_BAR_SIZE, region_type));
 
         let config = PciBarConfiguration::default()
-            .set_register_index(0)
+            .set_index(0)
```

Comment:
May worth adding an assignment to the constant `bar_idx` (e.g. `0` and `2`) more understandable? Not sure the following are the right naming: `let virtio_pci_bar_idx = 0;`, and `virtio_pci_shm_bar_idx = 2;`.

Reply:
Done